redfish-core/lib/log_services.hpp

```diff
@@ -6382,11 +6382,44 @@ inline void getRedfishUriByDbusObjPath(
         // Fill the all parents Redfish URI id.
         // For example, the processors id for the core.
         // "/redfish/v1/Systems/system/Processors/<str>/SubProcessors/core0"
+        std::vector<std::pair<RedfishResourceDBusInterfaces, size_t>>
+            ancestorsIfaces;
+        while (uriIdPos != std::string::npos)
+        {
+            std::string parentRedfishUri = redfishUri.substr(0, uriIdPos - 1);
+            RedfishUriListType::const_iterator parentRedfishUriIt =
+                std::find_if(redfishUriList.begin(), redfishUriList.end(),
+                             [parentRedfishUri](const auto& ele) {
+                return parentRedfishUri == ele.second;
+            });
```

Comment:
bmcweb prefers to use `std::ranges` like
```
            RedfishUriListType::const_iterator parentRedfishUriIt =
                std::ranges::find_if(redfishUriList, [parentRedfishUri](const auto& ele) {
                return parentRedfishUri == ele.second;
            });
```

Reply:
done